challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js

```diff
@@ -0,0 +1,25 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+console.log("---Exercise 01---");
+
+let num = 10;
+
+console.log("Even numbers between 10 and 40:");
+
+while (num <= 40) {
```

Comment:
These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones